### ISSUE TITLE
File path within Stack is incorrect when using `-f`

### ISSUE THREAD

**Issue by lukeed** [lukeed/taskr#181]
Using alternate flyfile path (`-f` option)

``` json
"scripts": {
    "fly": "fly -f web/static"
}
```

**Result:**

```
stack:
  - Error: File to read not found or unreadable: ~/.../web/static/app.sass
```

**Expected:**

```
stack:
  - Error: File to read not found or unreadable: ~/.../web/static/styles/app.sass
```

> Something to do with `this.root` and an incorrect `path.resolve` perhaps

**Comment by hzlmn** [comment]
@lukeed Are you working on it, or I can look on?

**Comment by lukeed** [comment]
@hzlmn You're more than welcome! Makes sense while you do #192

**Comment by lukeed** [comment]
Solved after #218
